src/components/Content/TaskHierarchy/Task/Task.js

```diff
@@ -167,7 +224,8 @@ class RawTask extends Component {
           <div className="toggle-ctn">
             {this.getTaskIcon(task, showChildren)}
           </div>
-          <div className="task-row-ctn">
+          {connectDropTarget(connectDragSource(
+          <div className="task-row-ctn" style={{ backgroundColor: canDrop && isOver ? '#50b3fe' : '' }}>
```

Comment:
For complicated security reasons we are trying to avoid inline styling. Could you replace this with something like `className={"task-row-ctn " + (canDrop && isOver ? 'highlight-task') : ''}` ?